examples/cartesian_reconstruction.py

```diff
@@ -0,0 +1,98 @@
+"""
+Neuroimaging cartesian reconstruction
+=====================================
+
+Author: Chaithya G R
+
+In this tutorial we will reconstruct an MRI image from the sparse kspace
+measurments.
+
+Import neuroimaging data
+------------------------
+
+We use the toy datasets available in pysap, more specifically a 2D brain slice
+and the acquisition cartesian scheme.
+"""
+
+# Package import
+from modopt.math.metrics import ssim
+from mri.numerics.reconstruct import sparse_rec_fista
+from mri.numerics.fourier import FFT2
+from mri.numerics.utils import generate_operators
+from mri.numerics.utils import convert_mask_to_locations
+import pysap
+
+# Third party import
+import numpy as np
+
+# Loading input data
+image = pysap.utils.load_image('../../../Data/Pysap_examples/base_image.npy')
```

Comment:
specify in a TODO that this is just temporary and needs fixing. Also specify it corresponds to the data we will retrospectively undersample

Reply:
Yes, I missed that.